wasm/__init__.py

```diff
@@ -1807,81 +1793,87 @@ def spec_reinterprett1t2(t1, t2, c):
 
 
 def spec_tconst(config):
-    if verbose >= 1:
-        print("spec_tconst(", ")")
+    logger.debug("spec_tconst()")
```

Comment:
Looks like it was helpful to include `c` from below in the logs. Maybe move this line down to include the value?